Real-Time Scheduled Search deprecation (May 15) (#4020)
SumoLogic/sumologic-documentation: blog-service/2024-05-15-scheduled-search-alerts.md

---
title: Deprecation Notice - Real-Time Scheduled Searches (Alerts)
image: https://www.sumologic.com/img/logo.svg
keywords:
  - alerts
  - monitors
  - scheduled search
hide_table_of_contents: true
authors:
  - url: https://help.sumologic.com/release-notes-service/rss.xml
    image_url: /img/release-notes/rss-orange.png
---

> Effective **May 15, 2024**, Real-Time Scheduled Searches have been deprecated and you can no longer create them. Real-Time Scheduled Searches created before that date will continue to function until **May 15, 2025**.

Reviewer: ```suggestion
As part of our ongoing evaluation of the Sumo Logic service, we have decided to deprecate [Real-Time Scheduled Searches](https://help.sumologic.com/docs/alerts/scheduled-searches/create-real-time-alert/). In particular, we will remove the option to create new Real-Time Scheduled Searches on May 15, 2024. Existing Real-Time Scheduled Searches will continue to function until May 15, 2025. We believe many use cases for Real-Time Scheduled Searches can be met by [Monitors](https://help.sumologic.com/docs/alerts/monitors/overview/). Any remaining use cases can be met by executing these searches at 15m intervals.
```

> We encourage you instead to [create a monitor](/docs/alerts/monitors/create-monitor) and set it to execute near-real-time searches (lowest available duration is 15m intervals).

Reviewer: ```suggestion
In 2020, Sumo Logic released Monitors, which provided a new framework to trigger alerts on both metrics and log data in real time and send notifications. Real-Time Scheduled Searches provided a much more limited version of this functionality, however Monitors will continue to be the focus area for our Product and Engineering Teams for features and enhancements regarding alerting.
```

> In 2020, Sumo Logic released Monitors, which provided a new framework to trigger alerts on both metrics and log data in real time and send notifications. Real-Time Scheduled Searches provided a much more limited version of this functionality, however Monitors will continue to be the focus area for our Product and Engineering Teams for features and enhancements regarding alerting.

Reviewer 2: ```suggestion
In 2020, Sumo Logic released Monitors, which provided a new framework to trigger alerts on both metrics and log data in real time and send notifications. Real-Time Scheduled Searches provided a much more limited version of this functionality. Monitors will continue to be the focus area for our Product and Engineering Teams for features and enhancements regarding alerting.
```
